docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -131,9 +132,9 @@ code: |
   if student.grade == "Not yet in school":
     student.grade = 'soon-to-be'
   else:
-    school.recipient.name = school.special_education_coordinator.name
-    school.recipient.address = school.special_education_coordinator.address
-    if not knows_special_education_director_name:
+    school.recipient.name = school.special_education_director_or_coordinator.name.first
```

Comment:
Remove .first from this line. It will make it so that only the first name shows on the form